tools/document-pii/src/main.rs

```diff
@@ -176,7 +176,8 @@ mod tests {
     #[test]
     fn test_find_rs_files() {
         let rust_crate = PathBuf::from_slash(RUST_TEST_CRATE);
-        let rust_file_paths = find_rs_files(&rust_crate);
+        let mut rust_file_paths = find_rs_files(&rust_crate);
+        rust_file_paths.sort_unstable();
```

Comment:
From [docs](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.sort):

>When applicable, unstable sorting is preferred because it is generally faster than stable sorting and it doesn’t allocate auxiliary memory. See [sort_unstable](https://doc.rust-lang.org/std/primitive.slice.html#method.sort_unstable).

Since there shouldn't be equal elements (it's files from the file system, in the end), I think unstable sorting is a better trade-off in this case.